build.sh

```diff
@@ -47,6 +47,8 @@ HELP="$0 [<target> ...] [<flag> ...]
    BACKEND_REF      - Commit ref for Triton backend repo when using build.py
    THIRDPARTY_REF   - Commit ref for Triton third-party repos when using build.py
    JOB_ID           - A unique id to use for this build job
+   USE_CLIENT_WHEEL - If 1, Triton client will be installed from wheel
```

Comment:
A little more documentation/definition of what/where the wheel comes from would be a good idea